plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestTrinoGlueCatalog.java

```diff
@@ -134,6 +149,57 @@ public void testNonLowercaseGlueDatabase()
         }
     }
 
+    @Test
+    public void testCreateMaterializedViewWithSystemSecurity()
+            throws IOException
+    {
+        Path rootPath = Files.createTempDirectory("test_iceberg_mv_with_glue");
+        TrinoCatalog glueTrinoCatalog = createGlueTrinoCatalog(false, true, new LocalFileSystemFactory(rootPath));
```

Comment:
HDFS_FILE_SYSTEM_FACTORY should work ok for local files, just use `file://` url
this will avoid adding `TrinoFileSystemFactory` parameter in `createTrinoCatalog` overload, keeping things simpler